### ISSUE TITLE
QA_EXPT-13 & EXPT-14_Procedure_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#41]
Defect Description :
In the Procedure page of 13th and 14th experiments of this lab, found that the content is same.

Actual Result :
In the Procedure page of 13th and 14th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i38](https://cloud.githubusercontent.com/assets/13479177/26666919/2833f60e-46c0-11e7-8ec2-9aa1b8ef4b68.png)

**Comment by sravanthimodepu** [comment]
Yes.. The13th & 14th experiments belongs to the same content but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
